schema/release-schema.json

```
@@ -1254,7 +1254,7 @@
         },
         "uri": {
           "title": "Linked budget information",
-          "description": "A URI pointing directly to a machine-readable record about the budget line-item or line-items that fund this contracting process. Information can be provided in a range of formats, including using IATI, the Open Fiscal Data Standard or any other standard which provides structured data on budget sources. Human readable documents can be included using the planning.documents block.",
+          "description": "A URI pointing directly to a machine-readable record about the budget line-item or line-items that will be used in the future contracting process. Information can be provided in a range of formats, including using IATI, the Open Fiscal Data Standard or any other standard which provides structured data on budget sources. Human readable documents can be included using the planning.documents block.",
```

The description of `Budget` had been changed to "according to the best available standards" in #1167. We forgot to remove the specific examples here.

```suggestion
          "description": "The URL of a machine-readable resource about the budget to be used in the future contracting process. For human-readable documents, see the `planning.documents` array.",
```